content/c-sharp/concepts/switch/switch.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Switch'
+Description: 'Switch statements execute a code block from a list of conditions if a condition is true.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Control Flow'
+  - 'Switch'
+  - 'C#'
```

Comment:
Can you remove the C# tag. We generally don't have language topics as Tags as they're redundant.

Reply:
Fixed